### PR TITLE
Centralize Gradle Test Setup

### PR DESCRIPTION
**Pull request by alvasw**
- [Remove redundant 'testRuntimeOnly libs.junit.jupiter.engine'](https://github.com/bisq-network/bisq/commit/509781acf28c0a0eef2754f91b001271b92381a7)
- [Remove redundant 'useJUnitPlatform()' declarations](https://github.com/bisq-network/bisq/commit/db6c1d1176946084cc082c2f5386076deb644221)

### CODE REVIEW

File: core/src/test/java/bisq/core/util/coin/CoinUtilTest.java [hunk 1]
```diff
@@ -88,9 +88,9 @@ public void testGetAdjustedAmount() {
                 20_000_000,
                 1);
         assertEquals(
-                "Minimum allowed trade amount should not be adjusted.",
+                result.toFriendlyString(),
                 "0.01 BTC",
```
Comment (napoly) [inline core/src/test/java/bisq/core/util/coin/CoinUtilTest.java]: sorry for the ripple effect here.. hard coded value should be expected and moved up by one line.. same goes to line 103

Comment (alvasw) [reply core/src/test/java/bisq/core/util/coin/CoinUtilTest.java]: Thank you!

**Comment by alejandrogarcia83** [comment]
Please rebase your PR to resolve the merge conflicts.

**Comment by alvasw** [comment]
Done

**Review by alejandrogarcia83 — APPROVED** [review]
utACK

**Pull request merged**